collections/ansible_collections/cloudkit/templates/roles/ocp_virt_vm/meta/cloudkit.yaml

```diff
@@ -0,0 +1,21 @@
+# Define the display name and description of the template
+title: OpenShift Virtualization VM
```

Comment:
nit: I'm not sure we want user to know it's running on OCP. We should just say that's a VM.